pykokkos/core/parsers/parser.py

```diff
@@ -123,19 +124,165 @@ def get_entities(self, style: PyKokkosStyles) -> Dict[str, PyKokkosEntity]:
 
         for i, node in enumerate(self.tree.body):
             if check_entity(node, self.pk_import):
-                start: int = node.lineno - 1
 
+                start: int = node.lineno - 1
                 try:
                     stop: int = self.tree.body[i + 1].lineno - 1
                 except IndexError:
                     stop = len(self.lines)
-
+                
                 name: str = node.name
+
                 entity = PyKokkosEntity(style, cppast.DeclRefExpr(name), node, (self.lines[start:stop], start), self.path, self.pk_import)
                 entities[name] = entity
 
         return entities
 
+
+    def fix_types(self, entity: PyKokkosEntity, updated_types: UpdatedTypes) -> ast.AST:
+
+        '''
+        updatedTypes: object that contains info about inferred types
+
+        This method will walk the ast for a workunit and add any mission annotations
+        This method will also invoke fix_viewlayouts that will add missing decorators for user specified layouts
+        '''
```

Comment:
Look at the docstrings for the other functions and use the same format

Reply:
You're still missing `:param` . For example, here it would be `:param entity: ...`. You're also missing the colon just before returns, so it should be `:returns:`. See the docstring [here](https://github.com/kokkos/pykokkos/blob/6b894e88b360fcaf21c152b739172ed0cc2ad013/pykokkos/core/compiler.py#L64).